backend/src/main/java/ai/verta/modeldb/configuration/RunLiquibaseSeparate.java

```diff
@@ -0,0 +1,36 @@
+package ai.verta.modeldb.configuration;
+
+import ai.verta.modeldb.common.CommonConstants;
+import java.util.Optional;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+public class RunLiquibaseSeparate implements Condition {
+  private static final Logger LOGGER = LogManager.getLogger(RunLiquibaseSeparate.class);
+
+  public RunLiquibaseSeparate() {}
```

Comment:
you don't need an explicit no-arg constructor, You get one for free from the compiler.

Reply:
Removed